bigbluebutton-tests/playwright/core/elements.js

```diff
@@ -34,7 +34,7 @@ exports.sendFeedbackButton = 'button[data-test="sendFeedbackButton"]';
 exports.feedbackCommentInput = 'textarea#feedbackComment';
 exports.errorScreenMessage = 'h1[data-test="errorScreenMessage"]';
 exports.errorMessageLabel = 'span[id="error-message"]';
-exports.shareCameraAsContent = 'li[data-test="shareCameraAsContent"]';
+exports.shareCameraAsContent = 'button[data-test="cameraAsContent"]';
```

Comment:
looks like correct selector should be `div`
<img width="1739" height="772" alt="image" src="https://github.com/user-attachments/assets/80928ddf-f216-4e52-9387-08ffb761fc8a" />

```suggestion
exports.shareCameraAsContent = 'div[data-test="cameraAsContent"]';
```